Source/Core/Core/HW/EXI/EXI.cpp

```diff
@@ -152,15 +152,15 @@ static void ChangeDeviceCallback(u64 userdata, s64 cyclesLate)
 }
 
 void ChangeDevice(const u8 channel, const TEXIDevices device_type, const u8 device_num,
-                  CoreTiming::FromThread from_thread)
+                  CoreTiming::FromThread from_thread, u32 delay)
```

Comment:
Perhaps `delay` should be a `std::chrono` type, so that the expected time unit is made evident in the interface?

Reply:
It's CPU cycles, actually, which we always just use an integer for. I've made it more clear.

Reply:
Though to be clear, I'm not opposed to adding a custom std::chrono type for CPU cycles, but that's probably for a different PR.